mbuild/tests/test_compound.py

```diff
@@ -68,6 +68,17 @@ def test_save_forcefield_with_file(self, methane):
                          forcefield_files=get_fn('methane_oplssaa.xml'),
                          overwrite=True)
 
+    @pytest.mark.skipif(not has_foyer, reason="Foyer is not installed")
+    def test_save_forcefield_with_file_foyerkwargs(self, methane):
+        exts = ['.gsd', '.hoomdxml', '.lammps', '.lmp', '.top', '.gro',
```

Comment:
I don't know if it's worth it to test _all_ these formats - probably testing this functionality can be accomplished just with a few well-behaved and well-supported ones (`.gro` alone would be sufficient, imo)

My concern is that this test would be a test over all the writers, which can cause unnecessary headaches in the future.